pkg/controllerhelpers/scylladbmanager.go

```diff
@@ -19,8 +19,7 @@ import (
 func GetScyllaDBManagerClient(_ context.Context, _ *scyllav1alpha1.ScyllaDBManagerClusterRegistration) (*managerclient.Client, error) {
 	url := fmt.Sprintf("http://%s.%s.svc/api/v1", naming.ScyllaManagerServiceName, naming.ScyllaManagerNamespace)
 	managerClient, err := managerclient.NewClient(url, func(httpClient *http.Client) {
-		// FIXME: https://github.com/scylladb/scylla-operator/issues/2693
-		httpClient.Transport = http.DefaultTransport
+		httpClient.Transport = http.DefaultTransport.(*http.Transport).Clone()
```

Comment:
I can see `GetScyllaDBManagerClient` is used only in finalizers so we can assume it won't be called very frequently, but I just wanted to surface calls to `Clone` _might_ degrade performance here. To preserve the status quo in terms of performance, we could have a global transport dedicated to manager clients, initialized only once.

Leaving the decision up to you, because most likely this is not critical and might be considered a premature optimization.